src/p2p/net_node.h

```diff
@@ -477,6 +477,11 @@ namespace nodetool
     extern const command_line::arg_descriptor<std::vector<std::string> > arg_p2p_add_priority_node;
     extern const command_line::arg_descriptor<std::vector<std::string> > arg_p2p_add_exclusive_node;
     extern const command_line::arg_descriptor<std::vector<std::string> > arg_p2p_seed_node;
+    extern const command_line::arg_descriptor<std::string> arg_p2p_ssl;
```

Comment:
I dropped those at vtnerd's request.

Reply:
Hmm, that sucks. I quite agree with your argument that encrypting everything by default is the right idea because it stops passive monitoring - which increases the resources needed to snoop.

Reply:
Well, it can be added later on if needed. vtnerd also pointed to some light weight encryption in Bitcoin for P2P, where we don't have to support all kinds of RPC clients like curl etc.